python/plotLHScans.py

```diff
@@ -487,4 +488,4 @@ def parseOptions():
                 f.write('resultsXS = '+str(resultsXS)+' \n')
             with open(datacardInputs+'/resultsXS_LHScan_observed_'+obsName.replace(' ','_')+'_v3.py', 'w') as f:
                 f.write('resultsXS = '+str(resultsXS_data)+' \n')
-    binCounter += 1
+        binCounter += 1
```

Comment:
was this a bug?

Reply:
yes, somehow it was out of loop and only printed the first bin's width on LScan plots for all bins.